messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -56,9 +57,16 @@ class MessagingTest : BaseMessagingTest() {
                             val dogId = dog.myId.id
                             val catId = cat.myId.id
 
+                            try {
+                                dog.addOrUpdateDirectContact("blub", "Bogus")
+                                fail("adding an invalid contact ID should fail")
+                            } catch (e: InvalidKeyException) {
+                                // okay
+                            }
+
                             val now = now
                             var catContact =
-                                dog.addOrUpdateDirectContact(catId, "Cat")
+                                dog.addOrUpdateDirectContact(catId.toUpperCase(), "Cat")
```

Comment:
toUpperCase allows me to break the ID to make sure it's getting sanitized properly